src/index.html

```diff
@@ -3,11 +3,393 @@
+          <p class="about-us__text">
+            Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical. <br><br>
+            The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists.
+            <br>Working directly with developers - so have more useful information about the profitable deals.
+            <br><br>
+            <big>We will never be late!<br>Guaranteed secure transaction - 100%</big>
+          </p>
+        </div>
+      </article>
+
+      <article class="gallery" id="gallery">
+        <h2 class="title-with-crown">
+          Gallery
+        </h2>
+        <section class="gallery__object">
+          <div class="gallery__images-row gallery__images-row--2-1">
+            <img
+              src="./images/photos/1.jpg"
+              alt="Marina Palms photo 1"
+              class="gallery__photo"
+            >
+            <img
+              src="./images/photos/2.jpg"
+              alt="Marina Palms photo 2"
+              class="gallery__photo"
+            >
+          </div>
+          <p class="gallery__object-descr">
+            <big class="bright-emphasis">Marina Palms </big>/ North Miami Beach, FL 33162
+          </p>
+        </section>
+
+        <section class="gallery__object">
+          <div class="gallery__images-row gallery__images-row--1-1">
+            <img
+              src="./images/photos/3.jpg"
+              alt="Edition Residens photo 1"
+              class="gallery__photo"
+            >
+            <img
+              src="./images/photos/4.jpg"
+              alt="Edition Residens photo 2"
+              class="gallery__photo"
+            >
+          </div>
+          <p class="gallery__object-descr">
+            <big class="bright-emphasis">Edition Residens </big>/ North Miami Beach, FL 33163
+          </p>
+        </section>
+
+        <section class="gallery__object">
+          <div class="gallery__images-row gallery__images-row--2-1">
+            <img
+              src="./images/photos/5.jpg"
+              alt="Faena House photo 1"
+              class="gallery__photo"
+            >
+            <img
+              src="./images/photos/6.jpg"
+              alt="Faena House photo 2"
+              class="gallery__photo"
+            >
+          </div>
+          <p class="gallery__object-descr">
+            <big class="bright-emphasis">Faena House </big>/ North Miami Beach, FL 33164
+          </p>
+        </section>
+
+        <input
+          type="checkbox"
+          id="photosHider"
+          class="gallery__checkbox"
+        >
+        <div class="gallery__hided">
+          <!-- VERY IMPORTANT!!! in scss variables change additionalPhotosCount-->
```

Comment:
remove this

Reply:
no! If you will add one more object or delete one object then it will not work correctly

Reply:
that's why you need to name your variables in css correctly, so that other developers will understand your code without comments

usually comments are not written for each such case, because you'll end up with another doc for code functionality

Reply:
dont you understand what means additionalPhotosCount?

Reply:
no, I can even guess what it could mean

anyway I meant your comment, it shouldn't be there

Reply:
you force me to shot in your foot. but ok, I deleted it